### DISCUSSION TITLE
Missing store in [@types/wordpress__notices]

### DISCUSSION THREAD

**Discussion by davilera** [DefinitelyTyped/DefinitelyTyped#59387] [Issues with a @types package]
`@wordpress/notices` exports a `store` variable ([at least since version 3.3.0](https://github.com/WordPress/gutenberg/blob/%40wordpress/notices%403.3.0/packages/notices/src/index.js)), but `@types/wordpress__notices` is missing said export.

[a comment by an automated account is not shown]

**Comment by chihsuan** [comment]
@davilera Thanks for reporting this! Would you be able to submit a PR for this?

**Comment by davilera** [comment]
Sure, @chihsuan, I can try.
